### PR TITLE
NIP-65: Relay list metada

### PR DESCRIPTION
**Pull request by verbiricha**

### CODE REVIEW

File: src/Element/Relays.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import "./Relays.css";
+import Nostrich from "nostrich.webp";
+import { useState } from "react";
+
+import Read from "Icons/Read";
+import Write from "Icons/Write";
+
+export interface RelaySpec {
```
Comment (v0l) [inline src/Element/Relays.tsx:8]: If this is part of NIP it should be in `Nostr/` dir?

File: src/Pages/settings/Relays.tsx [hunk 2]
```diff
@@ -20,6 +20,8 @@ const RelaySettingsPage = () => {
     const ev = await publisher.saveRelays();
     publisher.broadcast(ev);
     publisher.broadcastForBootstrap(ev);
+    const settingsEv = await publisher.saveRelaysSettings();
```
Comment (v0l) [inline src/Pages/settings/Relays.tsx:23]: Maybe we can write to all (or a lot of relays) from `https://api.nostr.watch/v1/online`

File: src/Element/Relays.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+interface RelaysProps {
+  relays: RelaySpec[];
+}
+
+const RelayFavicon = ({ url }: { url: string }) => {
+  const cleanUrl = url
+    .replace("wss://relay.", "https://")
+    .replace("wss://nostr.", "https://")
+    .replace("wss://", "https://")
```
Comment (v0l) [inline src/Element/Relays.tsx:21]: `.replace("ws://", "http://")`

**Review by v0l — COMMENTED** [review]
Maybe move `Relays.tsx` some other place since we already have `Relay.tsx`

File: src/Pages/settings/Relays.tsx (hunk 2 above)
Comment (verbiricha) [reply src/Pages/settings/Relays.tsx]: Sending to all of user relays and random sample of 20 from that endpoint.

**Review by v0l — APPROVED** [review]

**Pull request merged**